scala/scalafmt/scalafmt.bzl

```diff
@@ -0,0 +1,100 @@
+load(
```

Comment:
should this file be named `scalafmt_repositories`?

Reply:
It makes sense. Fixed.